packages/core/factories/__tests__/useUserFactory.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { useUserFactory } from '../src';
+
+const factoryParams = {
+  cart: jest.fn(),
+  loadUser: jest.fn(),
+  logOut: jest.fn(),
+  updateUser: jest.fn(),
+  register: jest.fn(),
+  logIn: jest.fn(),
+  changePassword: jest.fn()
+} as any;
+
+const useUserMethods = useUserFactory(factoryParams)();
+const consoleErrorSpy = jest.spyOn(console, 'error');
+describe('useUserFactory', () => {
```

Comment:
please take a look at describes pattern in factories, for example here should be
```suggestion
describe('[CORE - factories] useUserFactory', () => {
```
but go thru others as well. Good describing and naming tests is a great documentation later

Reply:
Done